src/lelantus.cpp

```diff
@@ -89,7 +89,7 @@ bool VerifyMintSchnorrProof(const uint64_t& v, const secp_primitives::GroupEleme
     return verifier.verify(comm, schnorrProof);
 }
 
-void ParseLelantusMintScript(const CScript& script, secp_primitives::GroupElement& pubcoin,  SchnorrProof& schnorrProof)
+void ParseLelantusMintScript(const CScript& script, secp_primitives::GroupElement& pubcoin,  SchnorrProof& schnorrProof, uint256& tagForRecover)
```

Comment:
sorry, most probably `mintTag` is better name. I can't quite understand why "recover" word is needed here